### ISSUE TITLE
Address with custom field creation not working

### ISSUE THREAD

**Issue by catokx** [bigcommerce/checkout-sdk-js#537]
I have 2 custom fields in customer addresses. If i palce an order with a "new" address with theese 2 custom fields set, bigcommerce creates the order with the custom fields in the order addresses, and it also creates the new customer address, but in the customer address the custom fields are empty.

**Comment by davidchin** [comment]
Thanks for reporting this issue. We are able to replicate this bug and currently looking for ways to fix it. We'll let you know once the fix is ready.

**Comment by fthobe** [comment]
Hi @davidchin, are there any updates to this issue?

**Comment by capsula4** [comment]
A fix for this has been deployed. Please feel free to open a new ticket if the issue has not been resolved for you.
